client/src/routes/westend/+page.ts

```diff
@@ -0,0 +1,10 @@
+import type { PageLoad } from "./$types";
+
+export const load: PageLoad<{ faq: string }> = async ({ fetch }) => {
+	const data = await fetch("/FAQ.md");
+	const textData = await data.text();
```

Comment:
Any particular reason not to include this on build and skip the fetch?

Reply:
Yeah, mainly to stop myself from installing 5 libraries that handle markdown parsing and import (too much boilerplate for a one particular case and I don't want to bloat the final js bundle).

By having it as a simple asset it is accesible and adds less loading time than an image.

Reply:
Wait, let me try something native

Reply:
It could be done easily and without any other libs or modifications! Brilliant!

Reply:
Nice one!